config/rewrites.json

```diff
@@ -12,6 +12,13 @@
             "to": "/$1/",
             "rewrite": false,
             "status": 301
+        },
+        {
+            "description": "Blog: Remove .html extension",
+            "from": "^\\/blog\\/(.+?)(\\.html)$",
```

Comment:
I'd be worried about it squashing legitimate html files if we apply it globally. In the spirit of not breaking stuff, I kinda want to start small.